xbmc/GUIInfoManager.cpp

```diff
@@ -5876,6 +5876,7 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
 ///                  _string_,
 ///     @return A concatenated string of cast members of the currently selected
 ///     movie\, for use in dialogvideoinfo.xml.
+///     @note Commas within the cast string will be replaced by carriage returns.
```

Comment:
Same for VideoPlayer.Cast, VideoPlayer.CastAndRole, ListItem.CastAndRole

Please add the comment also for these labels.

Reply:
@ksooo - I will make the request changes.

Reply:
Before I push the update, is this what you want to see?

![image](https://github.com/xbmc/xbmc/assets/127641886/b766305b-09f7-4e46-8fb9-ba6bc96ba588)

Reply:
> is this what you want to see?

Yes, thanks. That's it. Given you have updated all four affected labels, ListItem and VideoPlayer.